src/Resource/Ssl/Ssl.php

```diff
@@ -23,7 +23,10 @@ class Ssl extends Model {
   public const MODULE_NAME = 'Ssl';
 
   /** {@inheritDoc} */
-  protected const _PROPERTY_ALIASES = ['id' => 'cert_id'];
+  protected const _PROPERTY_ALIASES = [
+    'id' => 'cert_id',
+    'service_id' => 'service.id'
```

Comment:
Please confirm whether `service_id` is actually needed. I suspect it was an aberration from problems encountered during troubleshooting tests yesterday. I've ran unit tests and also made requests against demo2 without this param; everything seems to work fine.